packages/fast-components-styles-msft/src/slider/index.ts

```diff
@@ -0,0 +1,298 @@
+
+        slider__orientationVertical: {
+            "&$slider": {
+                minHeight: "128px",
+                minWidth: "32px",
+            },
+
+            "& $slider_thumb": {
+                justifySelf: "start",
+            },
+
+            "& $slider_layoutPanel": {
+                margin: "8px 0 8px 0",
+                gridTemplateColumns: "16px 1fr",
+            },
+
+            "& $slider_thumb_upper": {
+                transform: "translateY(-8px)",
+            },
+
+            "& $slider_thumb_lower": {
+                transform: "translateY(8px)",
+            },
+
+            "& $slider_barTrack": {
+                justifySelf: "start",
+                marginLeft: "6px",
+                width: "4px",
+                height: "100%",
+            },
+
+            "& $slider_barBack": {
+                justifySelf: "start",
+                marginLeft: "6px",
+                width: "4px",
+                top: "-2px",
+                bottom: "-2px",
+            },
+
+            "& $slider_barFront": {
+                justifySelf: "start",
+                marginLeft: "6px",
+                width: "4px",
+            },
+
+            "&$slider__modeAdjustLower": {
+                "& $slider_barFront": {
+                    marginTop: "-2px",
+                },
+            },
+
+            "&$slider__modeAdjustUpper": {
+                "& $slider_barFront": {
+                    marginBottom: "-2px",
+                },
+            },
+        },
+
+        slider__rtl: {
+            "&$slider__orientationHorizontal": {
+                "& $slider_thumb_upper": {
+                    transform: "translateX(-8px)",
+                },
+
+                "& $slider_thumb_lower": {
+                    transform: "translateX(8px)",
+                },
+
+                "&$slider__modeAdjustLower": {
+                    "& $slider_barFront": {
+                        marginRight: "0",
+                        marginLeft: "-2px",
+                    },
+                },
+
+                "&$slider__modeAdjustUpper": {
+                    "& $slider_barFront": {
+                        marginRight: "-2px",
+                        marginLeft: "0",
+                    },
+                },
+            },
+
+            "&$slider__orientationVertical": {
+                "& $slider_barBack": {
+                    marginRight: "6px",
+                    marginLeft: "0",
+                },
+
+                "& $slider_barFront": {
+                    marginRight: "6px",
+                    marginLeft: "0",
+                },
+            },
+        },
+
+        slider__modeSingle: {},
+
+        slider__modeAdjustLower: {},
+
+        slider__modeAdjustUpper: {},
+
+        slider__modeAdjustBoth: {
+            "&$slider__orientationHorizontal": {
+                "& $slider_thumb_upper": {
+                    width: "8px",
+                    borderTopRightRadius: "8px",
```

Comment:
Why not short hand border radius `borderRadius: "8px 0px 0px 8px"`

FYI for whatever reason with borderRadius `0` has to be followed by `px` unlike other properties like `margin` and `padding`.

Reply:
yep